tests/console/install_windowmanager.pm

```diff
@@ -0,0 +1,19 @@
+use base "consoletest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    become_root();
+    if (check_var("DESKTOP", "awesome")) {
+        assert_script_run("zypper -n in awesome");
+        script_run("sed -i 's/^DEFAULT_WM.*\$/DEFAULT_WM=\"awesome\"/' /etc/sysconfig/windowmanager");
+    }
+    script_run("exit");
```

Comment:
as I'm just replacing it - can you please avoid using script_run for exit - it's not a script, so just type_string "exit\n" please

(and yes, that's a pretty recent requirement)